zerver/lib/send_email.py

```diff
@@ -560,6 +560,15 @@ def custom_email_sender(
         #     vary user-to-user
         f.write(base_template.read().replace("{{ rendered_input }}", rendered_input))
 
+    # Add the manage_preferences block content in the plain_text template.
+    with (
+        open(plain_text_template_path, "a") as f,
+        open(
+            "templates/zerver/emails/custom_email_base.pre.manage_preferences_block.txt"
```

Comment:
This string should probably be defined as a variable, like with the other blocks in this function.

Reply:
Done.